perl-lib/OESS/lib/OESS/DB/User.pm

```diff
@@ -317,37 +317,50 @@ sub edit_user {
     return (undef, 'Required argument `email` is missing.') if !defined $email;
     return (undef, 'Required argument `auth_names` is missing.') if !defined $auth_names;
     return (undef, 'Required argument `status` is missing.') if !defined $status;
-     
+
     if ($given_name =~ /^system$/ || $family_name =~ /^system$/) {
         return(undef, "User 'system' is reserved.");
     }
 
+    my $query = "UPDATE user SET email = ?, given_names = ?, family_name = ?, status = ? WHERE user_id = ?";
 
-    my $query = "UPDATE user SET email = ?, given_names = ?, family_name = ?, status = ? WHERE user_id = $user_id";
-
-    my $results = $db->execute_query($query, [$email, $given_name, $family_name, $status]);
-
+    my $results = $db->execute_query($query, [$email, $given_name, $family_name, $status, $user_id]);
     if (!defined $user_id || $results == 0) {
         return (undef, "Unable to edit user - does this user actually exist?");
     }
 
-    # TODO Blindly removing and adding remote_auth entries for a user
```

Comment:
where did this come frome?

Reply:
branched from master instead of dev moved base branch